lib/hydra/works.rb

```diff
@@ -64,21 +64,38 @@ module Works
     autoload :GenerateThumbnail,                   'hydra/works/services/generic_file/generate/thumbnail'
     autoload :PersistDerivative,                   'hydra/works/services/generic_file/persist_derivative'
 
-    # model validations
-    def self.collection? collection
+    # model validations 
+    def self.works_collection? collection
```

Comment:
The test should be able to check

```ruby
works_gwork.works_generic_work?
```

instead of

```ruby
Hydra::Works.works_generic_work? works_gwork 
```

Does that work?